app/assets/javascripts/crop_upload.js

```diff
@@ -42,35 +44,45 @@ function add_cropper_handler(upload_path) {
       }
     }
   });
-  $modal.on('shown.bs.modal', function () {
-    cropper = new Cropper($image[0], {
+
+  modal.addEventListener('shown.bs.modal', function () {
+    cropper = new Cropper(image, {
       aspectRatio: aspectRatio,
       viewMode: 2,
     });
-  }).on('hidden.bs.modal', function () {
-    $input.val("");
+  });
+
+  modal.addEventListener('hidden.bs.modal', function () {
+    input.value = "";
     cropper.destroy();
     cropper = null;
   });
-  $('#crop').on('click', function () {
+
+  getById('crop').addEventListener('click', function () {
     let canvas;
-    let inputfile = $input.val().split('\\').pop();
-    $modal.modal('hide');
+    let inputfile = input.value.split('\\').pop();
+    const modalInstance = bootstrap.Modal.getInstance(modal);
+    modalInstance.hide();
```

Comment:
I feel like I'm starting to see this pattern with bootstrap modals often.  Do you think it makes sense to split this out as a helper method in `dom_utils.js` like `showOrCollapse`?